### ISSUE TITLE
Highlight on hover in Leaflet doesn't turn off

### ISSUE THREAD

**Issue by bebe2708** [tomchadwin/qgis2web#507]
Hi guys,
When a map is created with leaflet and you hover on a polygon it highlights in yellow but after you move the mouse it keeps it highlighted. This issue was not present in older versions of the plugin.
It still works if you export it with OpenLayers but in Leaflet the highlight doesn't turn off. 

![1](https://user-images.githubusercontent.com/22330327/27238381-8d622c84-52c4-11e7-832b-88f4d25dfca9.jpg)
![2](https://user-images.githubusercontent.com/22330327/27238382-8d668b76-52c4-11e7-9472-d537b373ffc5.jpg)

This code looks the same in older versions as well so it might be the leaflet plugin?

```
var highlightLayer;
        function highlightFeature(e) {
            highlightLayer = e.target;

            if (e.target.feature.geometry.type === 'LineString') {
              highlightLayer.setStyle({
                color: '#ffff00',
              });
            } else {
              highlightLayer.setStyle({
                fillColor: '#ffff00',
                fillOpacity: 1
              });
            }
        }

```

**Comment by tomchadwin** [comment]
I wonder if that is a side effect of the multi style plugin recently introduced. Can you make sure you have qgis2web 2.20.0, and style your layer with only one symbol layer? Let us know if that makes this bug disappear.

**Comment by bebe2708** [comment]
Thanks for the reply.
Yes I had 2.20.0 installed and I've tried like you said now, to style the layer with only one symbol layer but the bug still persists. In OpenLayers it works only with Leaflet it has this issue in all browsers.

**Comment by tomchadwin** [comment]
Confirmed. I think I understand the underlying issue - easy to fix for single symbol level styles, not sure about multi. I'll look at it ASAP.

**Comment by tomchadwin** [comment]
OK, I think it's fixed now for single-symbol-layer layers, but it's definitely not working for multiple symbol layers. Can you confirm?

**Comment by bebe2708** [comment]
Hero!
It works for single-symbol-layer
```
function pop_AOI0(feature, layer) {
            layer.on({
                mouseout: function(e) {
                    layer.setStyle(style_AOI0_0(feature));

                },
                mouseover: highlightFeature,
            });
```

**Comment by tomchadwin** [comment]
I just can't figure out how to get this to work with multi-symbol-layer symbols. Have to leave it for the moment. I've tied my head in knots trying to figure it out, but I just can't figure out a solution. I'd ask @perliedman, but it's a major feature creep from leaflet-multi-layer.

**Comment by tomchadwin** [comment]
Multi-style layer issue raised separately as #510.

**Comment by tomchadwin** [comment]
Released as 2.21.0 and pushed to QGIS plugins repo.

**Comment by tomchadwin** [comment]
Fix for multistyle committed to master. Please test.
